lib/fathead/bower/parse.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+
+URL_BASE = 'https://bower.io/docs/api/#'
+CODE_PATTERN = re.compile(
+    r'{% highlight sh %}(.*){% endhighlight %}',
+    re.DOTALL
+)
+
+items = []
+
+
+def add_item(ops):
+    # Write articles
+    items.append(
+     '\t'.join([
+        ops['name'],
+        'A',
+        '',
+        '',
+        '',
+        '',
+        '',
+        '',
+        '',
+        '',
+        '',
+        ops['abstract'],
+        URL_BASE + ops['URL']
+        ]))
+    # Write 'command' redirect for each article
+    items.append(
+     '\t'.join([
+        '%s command' % ops['name'],
+        'R',
+        ops['name'],
+        '',
+        '',
+        '',
+        '',
+        '',
+        '',
+        '',
+        '',
+        '',
+        ''
+        ]))
+
+def parse_item(text):
+    # remove all child block
+    text = text.split('\n####')[0]
+    name = text.split('\n')[0].strip()
+    code = re.search(CODE_PATTERN, text)
+    code = '<pre><code>%s</code></pre>' % (code.group(1).strip()) if code else ''
```

Comment:
@moollaza stripping the newlines from code here

![screen shot 2016-12-13 at 16 24 47](https://cloud.githubusercontent.com/assets/3740101/21148969/d5d73482-c151-11e6-9f30-f6a59eb71ff3.png)